clients/go/admin/token_source.go

```diff
@@ -27,10 +27,11 @@ func (ts CustomHeaderTokenSource) GetRequestMetadata(ctx context.Context, uri ..
 	}, nil
 }
 
-// Even though Admin is capable of serving authentication without SSL, we're going to require it here. That is, this module's
-// canonical Admin client will only do auth over SSL.
+// RequireTransportSecurity returns whether this credentials class requires TLS/SSL. OAuth uses Bearer tokens that are
+// susceptible to MITM (Man-In-The-Middle) attacks that are mitigated by TLS/SSL. We return false here to make it easier
+// to setup auth. However, in a production environment, TLS for Auth is a requirement.
 func (ts CustomHeaderTokenSource) RequireTransportSecurity() bool {
-	return true
+	return false
```

Comment:
Shouldn't this be configurable, trying to see how do we turn this on for production usecases as there checks in oauth that require this to be true on a secure channel.
eg : this check in credentials.go of grpc
```
credentials.CheckSecurityLevel(ri.AuthInfo, credentials.PrivacyAndIntegrity)
```

Reply:
done